helm/ffc-demo-web/values.yaml

```diff
@@ -19,7 +19,7 @@ container:
   runAsUser: 1000
   readOnlyRootFilesystem: true
   allowPrivilegeEscalation: false
-  apiGatewayUrl: http://ffc-demo-api-gateway.default
+  apiGatewayUrl: http://ffc-demo-api-gateway.ffc-demo
```

Comment:
The namespace change will break the Azure deployment, it's best to update the jenkins-aws file for now, we can move all the aws settings to `values.yaml` once we are off Azure.

Reply:
Ah that's a shame. Changed the default back and added the ffc-demo override to `jenkins-aws.yaml`.

I did the same thing in the API Gateway so I've fix that too: https://github.com/DEFRA/ffc-demo-api-gateway/pull/20.